setup.py

```diff
@@ -46,6 +46,23 @@
         "pytest-watch",  # `ptw` test watcher/runner
         "ipdb",  # Debugger (Must use `export PYTHONBREAKPOINT=ipdb.set_trace`)
     ],
+    # NOTE: These are extras that someone can install to get up and running quickly w/ ape
+    #       They should be kept up to date with what works and what doesn't out of the box
+    #       Usage example: `pipx install eth-ape[recommended-plugins]`
+    "recommended-plugins": [
```

Comment:
Maybe add a part to readme about this?

Reply:
Yeah, I was thinking to update the install guide. Could update README too